src/Core/Services/MetadataProviders/Converters/ObjectConverter.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure.DataApiBuilder.Core.Services.MetadataProviders.Converters
+{
+    /// <summary>
+    /// This is a converter to serialize and deserialize object type
+    /// can object can be of different types int, long, string, datetime. this converter handles different basic object types supported by
+    /// Hotchocolate
+    /// </summary>
+    public class ObjectConverter : JsonConverter<object>
```

Comment:
Do you not have to register this converter anywhere to ensure it is only used for `object`'s within the `DatabaseObject` class hierarchy? e.g. register like the converters in Config project: `RuntimeConfigLoader.cs` `GetSerializationOptions(...)`

Reply:
currently converters are in core and dataobject in config, we donot have access to core in config project, do you want me to move the converters to config folder instead ? 
I had in Metadataprovider folder as suggested earlier

Reply:
I'm not suggesting you move any files. 

I'm asking whether there is any "registration" process for the converters you wrote.
The pre-existing converters were "registered" via the following code in `RuntimeConfigLoader::GetSerializationOptions(...)`:
```csharp
JsonSerializerOptions options = new()
{
    PropertyNameCaseInsensitive = false,
    PropertyNamingPolicy = new HyphenatedNamingPolicy(),
    ReadCommentHandling = JsonCommentHandling.Skip,
    WriteIndented = true,
    DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
    Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping
};
options.Converters.Add(new EnumMemberJsonEnumConverterFactory());
options.Converters.Add(new RestRuntimeOptionsConverterFactory());
```

I don't see anything similar in this PR. Is registration not necessary?

Reply:
closing this out as discussed offline